### DISCUSSION TITLE
群晖挂载alist后的安全性--结贴

### DISCUSSION THREAD

**Discussion by githuburanus** [AlistGo/alist#3317] [Q&A]
群晖挂载alist后的安全性，外网只要在群晖的ip地址后加上5244，无需密码即可访问网盘内容，是不是太不安全了？
比如群晖在外网的域名为http://xxxxx.synology.me，只要加上5244端口即可远程访问。还是可以设置密码只是我不知道？

**Comment by githuburanus** [comment]
结贴，可以在元信息中设置各个网盘的访问密码

**Comment by githuburanus** [comment]
不过建议开发者能设计成没有输入密码前，连这个地址下的网盘目录都看不到

**Reply by wainyang** [reply]
设置访客路径或者禁用访客，这种简单问题不要复杂化了

**Comment by anwen-anyi** [comment]
alist.nn.ci 熟读文档
https://github.com/alist-org/alist/discussions/227 熟读这个顺便
